### PR TITLE
Troubleshooting Steps | Port Variable | Dockerfile for ARM64v8 | Build Docker From Source

### PR DESCRIPTION
**Pull request by shrunbr**
* Updated `README.md` to include troubleshooting steps that came out of #25.
* Updated `docker run` command in `README.md` to be in the correct format to be copied.
* Added a dockerfile for ARM64v8 architecture.
* Added `Building docker containers from source` section to `README.md` which includes steps for building a container for the `ARM64v8` architecture which is used by the Raspberry PI 4.
* Added `port: 8080` to `example.yml` config as discussed in #25.

### CODE REVIEW

File: Dockerfile-ARM64v8 [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM arm64v8/golang:1.15
```
Comment (shrunbr) [inline Dockerfile-ARM64v8:1]: Changed from to be arm64v8 based.

File: Dockerfile-ARM64v8 [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+COPY . /app
+ENV NODE_ENV=production
+ENV CGO_ENABLED=0
+RUN cd /app && \
+        test -e static.go || (echo "Please run go generate first"; exit 1) && \
+        go vet ./... && \
+        go test ./... && \
+        go build -o /tmp/patrol ./cmd/patrol
+
+FROM arm64v8/alpine:3.9
```
Comment (shrunbr) [inline Dockerfile-ARM64v8:11]: Changed from to be arm64v8 based.

File: README.md [hunk 3]
```diff
@@ -36,24 +36,51 @@ $ curl -sf https://gobinaries.com/karimsa/patrol/cmd/patrol | sh
...
+$ docker run -d \
+	--name patrol \
+	--restart=on-failure \
+	-v "$PWD:/data" \
+	-p 8080:8080 \
+	--log-driver json-file \
+	--log-opt max-size=100m \
+	ghcr.io/karimsa/patrol:latest \
+	run \
+	--config /config/patrol.yml
```
Comment (shrunbr) [inline README.md:48]: Fixed formatting.

File: README.md [hunk 4]
```diff
@@ -36,24 +36,51 @@ $ curl -sf https://gobinaries.com/karimsa/patrol/cmd/patrol | sh
...
+
+After you have cloned the repository you'll need to utilize go to compile/generate the working files from the source code. This is done by running `go generate` inside the directory where you cloned the repository. Think of `go generate` as the equivalent of `make`.
+
+Now that we have the code compiled and the prerequistes installed you can use `docker build` to create the containers. To generate a x86_64 architecture container run `docker build .`. This will use the file named `Dockerfile` to create the image.
+
+__If you're building ARM64v8...__
+
+To create an image for ARM64v8 architecture you'll need to specifiy the docker file in the `docker build` command. This can be done by running `docker build -f Dockerfile-ARM64v8 .` (yes include the period!). If you do not specify that specific file, when attempting to run the container on an ARM64v8-based system such as the Raspberry PI 4, you'll receive this error: `unable to prepare context: unable to evaluate symlinks in Dockerfile path`.
+
+
```
Comment (shrunbr) [inline README.md:83]: NEW - Build docker containers from source section

File: README.md [hunk 5]
```diff
@@ -158,6 +185,44 @@ $ SECRET=hello patrol run --config patrol.yml
...
+__Don't see your issue above?__
+
+When submitting an issue report please make sure to gather the docker container logs. This can be done by running `docker logs CONTAINER` on your docker host. Replace `CONTAINER` with the name you gave your patrol container. If you're using the docker run command above the command will be `docker logs patrol`. Make sure to copy and provide the output in the issue report.
+
+Please also provide your `patrol.yml` file. Make sure to sanitize it before submitting it so sensitive information isn't posted on Github. This includes but is not limited to names, addresses, phone numbers, public IP addresses (not [RFC1918](https://tools.ietf.org/html/rfc1918) or [RFC4193](https://tools.ietf.org/html/rfc4193) address space).
+
+Last but not least, anything else that you think will help diagnose your issue please include in the report as well.
+
+
```
Comment (shrunbr) [inline README.md:225]: NEW - Troubleshooting section that came out of #25

File: example.yml [hunk 6]
```diff
@@ -1,5 +1,6 @@
 name: MyApp Status
 db: data.db
+port: 8080
```
Comment (shrunbr) [inline example.yml:3]: Added port variable as discussed in #25.

File: Dockerfile-ARM64v8 (hunk 1 above)
Comment (karimsa) [reply Dockerfile-ARM64v8:1]: Hey! Could you change this to `1.15-alpine` - the `1.15` default is buster.

**Review by karimsa — APPROVED** [review]
I think this looks alright overall! I have a few changes I'd like to make, so I'll make those once it is merged in.

**Pull request merged**